packages/tiles/rollup.config.js

```diff
@@ -10,25 +9,8 @@ export default {
     css({
       output: 'all-components.css',
     }),
-    ts(),
-    swc({
```

Comment:
Had a realization that we’re running `tsc` anyway to build declarations; might as well just build the React components with 1 process instead of 2 (also they won’t ever disagree; SWC was a PITA to configure and it was giving me problems again)